src/cv/experience/Experience.js

```diff
@@ -16,38 +15,37 @@ class Experience extends Component {
             <div className="Cv-experience-text">
                 <img src={circleGreyMiddle} className="Cv-logo" alt="logo" />
                 <div className="Cv-experience-experience-title">{experience.position}</div>
-                {experience.mainProjects.map(function(project, index){
-                return (
-                    <div key={index}>
-                    <div className="Cv-experience-experience-desc">
-                        {project.desc}
-                    </div>
-                    <div className="Cv-experience-experience-technologies">
-                        Technologies used: {project.technologies.reduce((a, b) => {return a + ', ' + b})}
-                    </div>
-                    </div>
-                )
+                {experience.mainProjects.map(function (project, index) {
+                    return (
+                        <div key={index}>
+                            <div className="Cv-experience-experience-desc">
```

Comment:
Why 'experience-experience' ?

Reply:
fixed in d5fd54753086e85194a6e17253b7bf16c6b164fd